app/lib/f15080_vita_consent_to_disclose_pdf.rb

```diff
@@ -6,7 +6,7 @@ def source_pdf_name
   end
 
   def output_filename
-    "F15080 - VITA Consent To Disclose"
+    "F15080 - VITA Consent To Disclose.pdf"
```

Comment:
not feeling so super confident about this but without the extension the file type validator fails?